include/deal.II/lac/trilinos_block_sparse_matrix.h

```diff
@@ -246,6 +251,18 @@ namespace TrilinosWrappers
      */
     std::vector<Epetra_Map> range_partitioner () const DEAL_II_DEPRECATED;
 
+    /**
+     * Return the partitioning of the domain space of this matrix, i.e., the
+     * partitioning of the vectors this matrix has to be multiplied with.
+     */
+    std::vector<IndexSet> locally_owned_domain_indices() const;
```

Comment:
If you don't mind, can you add to this and the next function a sentence that explains that the returned index sets all correspond to the *index space of the individual blocks* of the matrix, not to the *global* index space of the overall matrix.